scripts/build.sh

```diff
@@ -14,22 +14,16 @@ case ${BUILD_CONFIG} in
     tar -zcvf deploy/EnergyPlusRegressionTool-"${VERSION_STRING}"-Ubuntu20.04.tar.gz -C dist runner
     ;;
 
-  Ubuntu18)
-    pyinstaller --onefile --add-data "epregressions/diffs/math_diff.config:epregressions/diffs" epregressions/runner.py
-    mkdir deploy
-    tar -zcvf deploy/EnergyPlusRegressionTool-"${VERSION_STRING}"-Ubuntu18.04.tar.gz -C dist runner
-    ;;
-
   Mac)
-    /Users/travis/Library/Python/3.7/bin/pyinstaller --onefile --noconsole --add-data "epregressions/diffs/math_diff.config:epregressions/diffs" epregressions/runner.py
+    pyinstaller --onefile --noconsole --add-data "epregressions/diffs/math_diff.config:epregressions/diffs" epregressions/runner.py
```

Comment:
I think eventually I'd like to just remove this script entirely, but it's OK for now, just cleaned it up a bit.